packages/table/src/common/loadableContent.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as React from "react";
+
+import { Classes } from "@blueprintjs/core";
+
+export interface ILoadingProps {
+    /**
+     * If true, show the component's loading state. When set on a component that supports
+     * `ICellProps`, content will be hidden behind a fixed-height skeleton. When set on all other
+     * components, children components that support `ICellProps` will display their loading state.
+     * @default false
```

Comment:
This interface will only be used on components that fall into these two categories. But having this indiscriminately in the docs feels weird. Better to just remove this then?